trio/_core/_run.py

```diff
@@ -445,9 +444,8 @@ def _add_exc(self, exc):
         self.cancel_scope.cancel()
 
     def _check_nursery_closed(self):
-        if (
-            not self._nested_child_running and not self._children
-            and not self._pending_starts
+        if not any(
+            [self._nested_child_running, self._children, self._pending_starts]
```

Comment:
Do you think this is an improvement in readability? Consistency-wise I don't think Trio ever uses `any()` over a literal list of bools and I personally avoid that style because I feel it's less expressive.

Reply:
Personally I do find it reads more naturally this way, as starting with "not any" makes reading the three attributes easier. It's also closer to English grammar! 

However, if there's any doubt or disagreement I'd rather take it out and get the rest merged.

Reply:
Eh, I *guess* I'd do it the way I originally did it? But they're both fine and let's focus on more important things :-)